backend/elasticsearch/src/main/java/com/rbmhtechnology/vind/elasticsearch/backend/util/SortUtils.java

```diff
@@ -147,4 +149,48 @@ protected static AggregationBuilder buildFacetSort(String name, Sort sort, Strin
                                 sort.getType()));
         }
     }
+
+    protected static AggregationBuilder buildSuggestionSort(String name,
```

Comment:
okay like this. another possible solution would be to create a `Sort.soBuilder(..)` in the Sort interface